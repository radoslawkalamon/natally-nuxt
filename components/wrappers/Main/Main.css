.main {
  background-color: var(--color-background-1);
  border-radius: 0 0 0 var(--border-radius);
  padding: var(--gap-section);
}

.main >>> > * {
  display: flex;
  flex-flow: column nowrap;
  gap: var(--gap-section);
  margin: 0 auto;
  max-width: var(--max-width-body);
}
