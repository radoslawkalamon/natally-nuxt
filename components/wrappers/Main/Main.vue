<template>
  <div class="main">
    <slot />
  </div>
</template>

<script src="./Main.ts" lang="ts"></script>
<style src="./Main.css" scoped></style>
