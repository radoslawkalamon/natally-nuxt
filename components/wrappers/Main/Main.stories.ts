import { Meta, Story } from '@storybook/vue'
import Main from './Main.vue'

const meta: Meta = {
  title: 'Wrappers/Main',
  component: Main,
  argTypes: {
    default: {
      description: '-',
      table: {
        type: null,
        defaultValue: null
      }
    }
  }
}
export default meta

export const Default: Story = () => ({
  components: { Main },
  template: '<Main>Default Slot</Main>'
})
