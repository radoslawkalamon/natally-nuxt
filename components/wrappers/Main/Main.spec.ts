import Main from './Main.vue'
import { shallHaveSlot, shallRender } from '@/devtools/jest.shared.spec'

describe('Wrappers / Main', () => {
  shallRender(Main)
  shallHaveSlot(Main, 'default')
})
