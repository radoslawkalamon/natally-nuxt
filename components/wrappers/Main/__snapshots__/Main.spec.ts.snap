// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrappers / Main Unit shall pass unit sanity test 1`] = `"<main class="main">Slot default text</main>"`;
